BSP/STM32F413/Src/BSP_ADC.c

```diff
@@ -71,15 +80,20 @@ void BSP_ADC_Init(void) {
 	ADC_InitStruct.ADC_ExternalTrigConvEdge = ADC_ExternalTrigConvEdge_None;
 	ADC_InitStruct.ADC_ExternalTrigConv = DISABLE;
 	ADC_InitStruct.ADC_DataAlign = ADC_DataAlign_Right;
-	ADC_InitStruct.ADC_NbrOfConversion = 2;							// We have two channels that we need to read
+	ADC_InitStruct.ADC_NbrOfConversion = NUMBER_OF_CHANNELS;							// We have four channels (including extras) that we need to read
 
 	ADC_Init(ADC1, &ADC_InitStruct);
 
 	// Configure the channels
 	// Apparently channel 2 has priority, or is at least read first.
 	// If you change the priorities, be prepared to have the order in the array change.
-	ADC_RegularChannelConfig(ADC1, ADC_Channel_10, 1, ADC_SampleTime_480Cycles);	// Accelerator
-	ADC_RegularChannelConfig(ADC1, ADC_Channel_11, 2, ADC_SampleTime_480Cycles);	// Brake
+    ADC_RegularChannelConfig(ADC1, ExtraADC_1_CHANNEL,0, ADC_SampleTime_480Cycles); // Extra 1
+    ADC_RegularChannelConfig(ADC1, ExtraADC_2_CHANNEL,1, ADC_SampleTime_480Cycles); // Extra 2
+	ADC_RegularChannelConfig(ADC1, ACCEL_POT_CHANNEL, 2, ADC_SampleTime_480Cycles);	// Accelerator - Used to be priority 1, now 2
+	ADC_RegularChannelConfig(ADC1, BRAKE_POT_CHANNEL, 3, ADC_SampleTime_480Cycles);	// Brake - Used to be priority 2, now 3
+    // (TODO) HELLLLP... I kept the extra ADCs at lower numbers assuming priorities are higher the bigger the number?
```

Comment:
I'm not completely certain, but according to [this](https://visualgdb.com/tutorials/arm/stm32/adc/), I think lower numbers = higher rank = higher priority